syntax = "proto3";

package tensorflow.serving;
option cc_enable_arenas = true;

import "tensorflow_serving/apis/model.proto";
import "tensorflow_serving/config/logging_config.proto";

// Metadata logged along with the request logs.
message LogMetadata {
  ModelSpec model_spec = 1;
  SamplingConfig sampling_config = 2;
  // TODO(b/33279154): Add more metadata as mentioned in the bug.
}
